content/term/opensource.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "Open Source"
+title:
+- "Le riche écosystème de l'Open Source"
+titleSeo: "Contribuez à l'Open Source à travers les projets que vous confiez à Elao"
+metaDescription : "L'Open Source reposant sur le principe du projet communautaire, évolue dans un écosystème riche et impactant. Découvrez notre contribution chez Elao. "
+articles:
+---
+
+L'Open Source, reposant sur le principe du projet communautaire dénombre de grands avantages. Chez Elao, cette philosophie est au cœur de notre culture, nous encourageant à contribuer et à partager de nombreux produits Open Source.
+
+## L'Open Source, qu'est-ce que c'est ?
+
+Un logiciel Open Source est un logiciel dont le code est mis à disposition d'une manière **ouverte**, **publique**. Chacun peut accéder au développement et récupérer le code au besoin.
+
+Ce principe, permet aux créateur du code de partager leur travail et d'apporter une aide aux autres développeurs. Puisque chacun peut s'approprier le code, les bugs qui peuvent y êtres présents sont rapidement détectés par la communauté qui l'utilise, et donc corrigés facilement.
+
+L'inconvénient ? Rien ne garanti que le logiciel, le code, sera développé rapidement. Le créateur peut ne pas le mettre à jour ou le faire évoluer, de fait, il deviendra obsolète.
+
+L'Open Source est dans notre quotidien, sans que l'on s'en rende compte : LibreOffice, GIMP ou encore Github pour ne citer qu'eux.
+
+## Elao et l'Open Source
+
+Chez Elao, ce principe de l'Open Source est très présent et fait partie intégrante de notre **ADN**.  
+Nous considérons cet **écosystème** comme une ressource à laquelle nous contribuons tout en en bénéficiant, que ce soit en l'intégrant dans nos projets pour des fonctionnalités spécifiques ou en partageant nos propres produits pour le profit de tous. 
+C'est pourquoi, nous mettons à disposition de nombreux produits made in Elao ! 
```

Comment:
Suggest : On en a pas tant que ça en vrai, et le temps a manqué ces derniers temps à ceux qui travaillaient sur ces projets.
```suggestion
```

Reply:
On peut peut être reformuler ainsi, qu'en penses-tu ? "C'est pourquoi, nous avons pour volonté de mettre à disposition des produits made in Elao !".

Reply:
Sinon au lieu de les lister, on peut indiquer le compte GH elao (et manala ?), les repo public sont par principe accessibles aux personnes exterieures à l'orga. On y retrouve les projets open source du coup, serieux et fun.
Je me dis que ça permettra à la page de mieux veillir. Si on liste des contrib qu'on abandonne, je pense pas qu'on ait à l'esprit de venir éditer cette page

***

Retrouvez sur notre [compte GitHub Elao](https://github.com/Elao) ainsi que sur celui de [Manala](https://github.com/manala) les différentes contributions open source des membres de l'équipe.